getting-started.html

```diff
@@ -0,0 +1,329 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="description" content="">
+    <meta name="author" content="">
+    <!-- not working -->
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
+
+    <title>LoopBack 4</title>
+
+    <!-- Bootstrap core CSS -->
+    <link href="dist/bootstrap/css/bootstrap.min.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=IBM+Plex+Sans" rel="stylesheet">
+    <link href="css/loopback.css" rel="stylesheet">
+    <link href="css/getting-started.css" rel="stylesheet">
+
+    <!-- Site Analytics -->
+    <script type="text/javascript">
+        /* Define digital data object based on _appInfo object */
+        window.digitalData = {
+            page: {
+                category: {
+                    primaryCategory: 'IBM_Cloud_GlobalMarketing',
+                },
+                pageInfo: {
+                    ibm: {
+                        siteID: 'Cloud',
+                    }
+                },
+            }
+        };
+    </script>
+    <script src="//1.www.s81c.com/common/stats/ida_stats.js" type="text/javascript"></script>
+</head>
+
+<body>
+    <!-- Navbar -->
+    <nav class="navbar navbar-expand-lg navbar-light fixed-top">
+        <a class="navbar-brand" href="#">
+            <img src="img/global/loopback-full-logo-blue.svg" width=150px>
+        </a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent"
+            aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+            <ul class="navbar-nav ml-auto">
+                <li class="nav-item">
+                    <a class="nav-link" href="getting-started.html">Getting Started</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="https://github.com/strongloop/loopback-next/blob/master/docs/CONTRIBUTING.md">Contribute</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="http://loopback.io/doc/en/lb4">Docs</a>
+                </li>
+                <li class="nav-item" style="padding-top: 10px">
+                    <a class="github-button" href="https://github.com/strongloop/loopback-next" data-icon="octicon-star" data-show-count="true"
+                        aria-label="Star strongloop/loopback-next on GitHub">Star</a>
+                </li>
+
+            </ul>
+
+        </div>
+    </nav>
+    <!-- Header -->
+    <div class="gs-masthead">
```

Comment:
This background image needs to be muted -- spec seems to have an opacity overlay from top to bottom which gives better readability of the text on this (especially the smaller second line). Please implement that!

Reply:
I'll change the opacity from 0.4 to 0.2 in the new version.  Hope it's better.

Reply:
got it